arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/popup/viewmodel/PopupViewModel.kt

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright 2021 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.popup.viewmodel
```

Comment:
For the benefit of less nesting, I suggest to put all popup-related types in a single package without further sub-packages:
```suggestion
package com.esri.arcgisruntime.toolkit.popup
```

Reply:
updated